content/features/featuresDeepDive/importers/createImporters.md

```diff
@@ -12,37 +12,97 @@ video-content:
 
 By default, babylon.js comes with an importer for .babylon files.
 
-You can also create your own importer by providing a specific object to the `BABYLON.SceneLoader.RegisterPlugin` function.
+You can also create your own importer by providing a specific object to the `BABYLON.registerSceneLoaderPlugin` function.
 
-This object must have three properties:
+### Plugins
 
-- A list of supported file extensions (`extensions`)
-- An `importMesh` function to import specific meshes
-- A `load` function to import complete scene
-- A `loadAssets` function to import all babylon elements from the file but do not add them to the scene
+A file importer should implement the `ISceneLoaderPluginAsync` interface.
 
-Here is a sample importer object:
+<Alert severity="warning" title="Warning" description="Avoid using ISceneLoaderPlugin as it is legacy and has been replaced by ISceneLoaderPluginAsync" />

```

Comment:
```suggestion
<Alert severity="warning" title="Warning" description="Avoid using ISceneLoaderPlugin as it is deprecated and has been replaced by ISceneLoaderPluginAsync" />
```

feels clearer? even though it is not deprecated in code, it makes more sense when reading it